protractor-docker/docker-selenium-grid-conf.js

```diff
@@ -0,0 +1,19 @@
+exports.config = {
+	framework: 'jasmine2'
+	, specs: ['specs/*.js']
+ 	, seleniumAddress: 'http://localhost:4444/wd/hub'
+	,  capabilities: {
+		browserName: 'chrome'
+		, shardTestFiles: true
+		, maxInstances: 5
```

Comment:
@cnishina  We need so that we can run the 5 spec files in parallel at the same time. Without all the tests run on one single node. Checked it. Will mention it.